components/basket/Summary.tsx

```diff
@@ -9,24 +9,25 @@ const ClientInfo: React.FC<{ client: Client }> = ({ client }) => (
     </div>
     <div className="tw-text-gray-800">
       <span>{client.addressLine1}</span>
-      {client.addressLine1 && <div>{client.addressLine1}</div>}
+      {client.addressLine2 && <div>{client.addressLine2}</div>}
     </div>
     <div className="tw-text-gray-800">
       {client.city}, {client.postal}
     </div>
   </div>
 );
 
-export const Summary: React.FC<{ items: LineItem[]; client?: Client }> = ({
-  items,
-  client,
-}) => {
+export const Summary: React.FC<{
+  items: LineItem[];
+  client?: Client;
+  onOrder: () => void;
+}> = ({ items, client, onOrder }) => {
   return (
     <div>
       <ClientInfo client={client} />
       <Cart items={items} />
       <div className="tw-my-4 tw-mx-6 tw-flex tw-justify-center">
-        <Button type="primary" shape="round">
+        <Button type="primary" shape="round" onClick={() => onOrder()}>
```

Comment:
`onClick={onOrder}` won't work?

Reply:
Would work but TS is complaining about proper event types 🤮